modules/backend/widgets/Lists.php

```diff
@@ -1154,6 +1155,18 @@ protected function evalCustomListType($type, $record, $column, $value)
         throw new ApplicationException(sprintf('List column type "%s" could not be found.', $type));
     }
 
+    /**
+     * Process as selectable value types for 'dropdown', 'radio', 'checkboxlist', 'balloon-selector' form field types.
+     */
+    protected function evalSelectableTypeValue($record, $column, $value)
+    {
+        $fieldName = $column->columnName;
+        $fieldOptions = $column->config['options'] ?? null;
+        $fieldOptions = $this->getOptionsFromModel($record, $fieldOptions, $fieldName);
```

Comment:
@SaifurRahmanMohsin You could maybe condense this down to one method call and remove the two lines above it?
```suggestion
        $fieldOptions = $this->getOptionsFromModel(
            $record,
            $column->config['options'] ?? null,
            $column->columnName
        );
```